WooCommerce/WooCommerceTests/ViewModels/OrderListCellViewModelTests.swift

```diff
@@ -0,0 +1,56 @@
+import XCTest
+import TestKit
+
+@testable import Yosemite
```

Comment:
⚠️ This doesn't need to be `@testable`. 🙂 

```suggestion
import Yosemite
```

It seems to compile without it. 🤪